BUILD.md

```diff
@@ -122,7 +122,7 @@ to deploy it as part of KEDA. Do the following:
     kubectl logs -l app=keda-metrics-apiserver -n keda -f
     ```
 
-## Debugging
+## Debugging Operator
```

Comment:
Nit - We might want to use h2 for "Debugging" and have h3 per operator/metric server

Reply:
PTAL